zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/config/ExporterConfiguration.java

```diff
@@ -131,6 +131,10 @@ public static final class IndexSettings {
 
     private Integer variableSizeThreshold = DEFAULT_VARIABLE_SIZE_THRESHOLD;
 
+    private boolean shouldWaitForImporters = true;
+
+    private Integer maxCachedRecordsBeforeErrorWhileWaitingForImporters = 1_000;
```

Comment:
❓ can we re-use the `bulk.size` here instead of creating a new config value?

Reply:
Sure I don't see any harm just wanted to make sure the reasoning was clear

Reply:
I understand. My feeling is this is temporary, and it makes sense to stop at the configured bulk.size anyway. We can make it clear through documentation for the 8.7 version

Reply:
Sure I don't see a problem